### PR TITLE
Fix Lots of Error: file_exists(): open_basedir restriction in effect.…

### LINKED ISSUE #27759: Lots of Error: file_exists(): open_basedir restriction in effect. in Log
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Steps to reproduce
1. Login to Nextcloud
2. Go to Settings -> Logs
3. Be annoyed by the messages

### Expected behaviour
No open_basedir restriction messages.

### Actual behaviour
Error: file_exists(): open_basedir restriction in effect messages for the following paths: 
/appinfo/routes.php
/l10n/en.js
/templates/
/img/app.svg (and lots of other SVGs)

### Server configuration

**Operating system:** Ubuntu 20.04

**Web server:** Apache2 behind Nginx reverse proxy

**Database:** MariaDB 10.5.11

**PHP version:** 8.0.7

**Nextcloud version:** 21.0.3

**Updated from an older Nextcloud/ownCloud or fresh install:** Updated

**Where did you install Nextcloud from:** nextcloud.com

**Signing status:**
<details>
<summary>Signing status</summary>
No errors have been found.
</details>

**List of activated apps:**
<details>
<summary>App list</summary>
Enabled:
  - accessibility: 1.7.0
  - activity: 2.14.3
  - admin_audit: 1.11.0
  - apporder: 0.13.0
  - breezedark: 21.0.8
  - camerarawpreviews: 0.7.11
  - checksum: 1.1.2
  - cloud_federation_api: 1.4.0
  - comments: 1.11.0
  - contactsinteraction: 1.2.0
  - dashboard: 7.1.0
  - dav: 1.17.1
  - documentserver_community: 0.1.9
  - extract: 1.3.1
  - federatedfilesharing: 1.11.0
  - federation: 1.11.0
  - files: 1.16.0
  - files_antivirus: 3.2.1
  - files_downloadactivity: 1.10.0
  - files_external: 1.12.0
  - files_fulltextsearch: 21.0.2
  - files_mindmap: 0.0.24
  - files_pdfviewer: 2.1.0
  - files_rightclick: 1.0.0
  - files_sharing: 1.13.1
  - files_texteditor: 2.14.0
  - files_trackdownloads: 1.10.0
  - files_trashbin: 1.11.0
  - files_versions: 1.14.0
  - files_videoplayer: 1.10.0
  - firstrunwizard: 2.10.0
  - fulltextsearch: 21.0.1
  - fulltextsearch_elasticsearch: 21.0.1
  - imageconverter: 1.3.1
  - impersonate: 1.8.0
  - logreader: 2.6.0
  - lookup_server_connector: 1.9.0
  - mail: 1.9.5
  - maps: 0.1.8
  - metadata: 0.13.0
  - notifications: 2.9.0
  - notify_push: 0.2.1
  - oauth2: 1.9.0
  - onlyoffice: 7.0.2
  - password_policy: 1.11.0
  - passwords: 2021.6.20
  - photos: 1.3.0
  - previewgenerator: 3.1.1
  - privacy: 1.5.0
  - provisioning_api: 1.11.0
  - ransomware_protection: 1.10.1
  - recommendations: 1.0.0
  - serverinfo: 1.11.0
  - settings: 1.3.0
  - sharebymail: 1.11.0
  - spreed: 11.2.2
  - support: 1.4.0
  - survey_client: 1.9.0
  - systemtags: 1.11.0
  - text: 3.2.0
  - theming: 1.12.0
  - twofactor_backupcodes: 1.10.0
  - updatenotification: 1.11.0
  - user_ldap: 1.11.0
  - user_status: 1.1.1
  - viewer: 1.5.0
  - weather_status: 1.1.0
  - workflowengine: 2.3.0
Disabled:
  - encryption
  - nextcloud_announcements
</details>

**Nextcloud configuration:**
<details>
<summary>Config report</summary>

{
    "system": {
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "cloud.clnp.eu",
            "cloud.saef-zf.de"
        ],
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "mysql",
        "version": "21.0.3.1",
        "overwrite.cli.url": "https:\/\/cloud.clnp.eu",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "mysql.utf8mb4": true,
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "mail_smtpmode": "sendmail",
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "memcache.local": "\\OC\\Memcache\\APCu",
        "memcache.distributed": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "***REMOVED SENSITIVE VALUE***",
            "port": 0
        },
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "updater.release.channel": "stable",
        "loglevel": 0,
        "maintenance": false,
        "theme": "",
        "app_install_overwrite": [
            "camerarawpreviews",
            "files_texteditor",
            "extract",
            "documentserver_community"
        ],
        "ldapIgnoreNamingRules": false,
        "ldapProviderFactory": "OCA\\User_LDAP\\LDAPProviderFactory",
        "default_language": "de",
        "default_locale": "de",
        "default_phone_region": "DE",
        "defaultapp": "files",
        "trusted_proxies": "***REMOVED SENSITIVE VALUE***"
    }
}
</details>

**Are you using external storage, if yes which one:** Local, Nextcloud

**Are you using encryption:** no

**Are you using an external user-backend, if yes which one:** no

### Client configuration
**Browser:** Firefox/Chrome/Edge, does not matter

**Operating system:** Windows 10, still does not matter

#### Web server error log
<details>
<summary>Web server error log</summary>
nothing relevant there. Just 200s from log polling
</details>

#### Nextcloud log (data/nextcloud.log)
<details>
<summary>Nextcloud log</summary>
Error	PHP	Error: file_exists(): open_basedir restriction in effect. File(/templates/) is not within the allowed path(s): (/var/www/vhosts/domain.tld/:/tmp/:/proc/meminfo:/proc/cpuinfo:/proc/uptime:/usr/share/dict/words) at /var/www/vhosts/domain.tld/cloud.domain.tld/lib/private/Template/Base.php#68	
Gerade eben
Error	PHP	Error: file_exists(): open_basedir restriction in effect. File(/appinfo/routes.php) is not within the allowed path(s): (/var/www/vhosts/domain.tld/:/tmp/:/proc/meminfo:/proc/cpuinfo:/proc/uptime:/usr/share/dict/words) at /var/www/vhosts/domain.tld/cloud.domain.tld/lib/private/Route/Router.php#135	
Gerade eben
Error	PHP	Error: file_exists(): open_basedir restriction in effect. File(/appinfo/routes.php) is not within the allowed path(s): (/var/www/vhosts/domain.tld/:/tmp/:/proc/meminfo:/proc/cpuinfo:/proc/uptime:/usr/share/dict/words) at /var/www/vhosts/domain.tld/cloud.domain.tld/lib/private/Route/Router.php#135	
Gerade eben
Error	PHP	Error: file_exists(): open_basedir restriction in effect. File(/appinfo/routes.php) is not within the allowed path(s): (/var/www/vhosts/domain.tld/:/tmp/:/proc/meminfo:/proc/cpuinfo:/proc/uptime:/usr/share/dict/words) at /var/www/vhosts/domain.tld/cloud.domain.tld/lib/private/Route/Router.php#135	
Gerade eben
Error	PHP	Error: file_exists(): open_basedir restriction in effect. File(/appinfo/routes.php) is not within the allowed path(s): (/var/www/vhosts/domain.tld/:/tmp/:/proc/meminfo:/proc/cpuinfo:/proc/uptime:/usr/share/dict/words) at /var/www/vhosts/domain.tld/cloud.domain.tld/lib/private/Route/Router.php#135	
Gerade eben
Error	PHP	Error: file_exists(): open_basedir restriction in effect. File(/templates/) is not within the allowed path(s): (/var/www/vhosts/domain.tld/:/tmp/:/proc/meminfo:/proc/cpuinfo:/proc/uptime:/usr/share/dict/words) at /var/www/vhosts/domain.tld/cloud.domain.tld/lib/private/Template/Base.php#68	
Gerade eben
Error	PHP	Error: file_exists(): open_basedir restriction in effect. File(/appinfo/routes.php) is not within the allowed path(s): (/var/www/vhosts/domain.tld/:/tmp/:/proc/meminfo:/proc/cpuinfo:/proc/uptime:/usr/share/dict/words) at /var/www/vhosts/domain.tld/cloud.domain.tld/lib/private/Route/Router.php#135	
Gerade eben
Error	PHP	Error: is_file(): open_basedir restriction in effect. File(/l10n/en.js) is not within the allowed path(s): (/var/www/vhosts/domain.tld/:/tmp/:/proc/meminfo:/proc/cpuinfo:/proc/uptime:/usr/share/dict/words) at /var/www/vhosts/domain.tld/cloud.domain.tld/lib/private/Template/ResourceLocator.php#106	
Gerade eben
Error	PHP	Error: file_exists(): open_basedir restriction in effect. File(/templates/) is not within the allowed path(s): (/var/www/vhosts/domain.tld/:/tmp/:/proc/meminfo:/proc/cpuinfo:/proc/uptime:/usr/share/dict/words) at /var/www/vhosts/domain.tld/cloud.domain.tld/lib/private/Template/Base.php#68	
Gerade eben
Error	PHP	Error: file_exists(): open_basedir restriction in effect. File(/templates/) is not within the allowed path(s): (/var/www/vhosts/domain.tld/:/tmp/:/proc/meminfo:/proc/cpuinfo:/proc/uptime:/usr/share/dict/words) at /var/www/vhosts/domain.tld/cloud.domain.tld/lib/private/Template/Base.php#68	
Gerade eben
Error	PHP	Error: file_exists(): open_basedir restriction in effect. File(/img/app.svg) is not within the allowed path(s): (/var/www/vhosts/domain.tld/:/tmp/:/proc/meminfo:/proc/cpuinfo:/proc/uptime:/usr/share/dict/words) at /var/www/vhosts/domain.tld/cloud.domain.tld/lib/private/Template/IconsCacher.php#132	
vor 2 Minuten
Error	PHP	Error: file_exists(): open_basedir restriction in effect. File(/img/circles.svg) is not within the allowed path(s): (/var/www/vhosts/domain.tld/:/tmp/:/proc/meminfo:/proc/cpuinfo:/proc/uptime:/usr/share/dict/words) at /var/www/vhosts/domain.tld/cloud.domain.tld/lib/private/Template/IconsCacher.php#132	
vor 2 Minuten
Error	PHP	Error: file_exists(): open_basedir restriction in effect. File(/img/column.svg) is not within the allowed path(s): (/var/www/vhosts/domain.tld/:/tmp/:/proc/meminfo:/proc/cpuinfo:/proc/uptime:/usr/share/dict/words) at /var/www/vhosts/domain.tld/cloud.domain.tld/lib/private/Template/IconsCacher.php#132	
vor 2 Minuten
Error	PHP	Error: file_exists(): open_basedir restriction in effect. File(/img/options.svg) is not within the allowed path(s): (/var/www/vhosts/domain.tld/:/tmp/:/proc/meminfo:/proc/cpuinfo:/proc/uptime:/usr/share/dict/words) at /var/www/vhosts/domain.tld/cloud.domain.tld/lib/private/Template/IconsCacher.php#132	
vor 2 Minuten
Error	PHP	Error: file_exists(): open_basedir restriction in effect. File(/img/filteradd.svg) is not within the allowed path(s): (/var/www/vhosts/domain.tld/:/tmp/:/proc/meminfo:/proc/cpuinfo:/proc/uptime:/usr/share/dict/words) at /var/www/vhosts/domain.tld/cloud.domain.tld/lib/private/Template/IconsCacher.php#132	
vor 2 Minuten
Error	PHP	Error: file_exists(): open_basedir restriction in effect. File(/img/emoji.svg) is not within the allowed path(s): (/var/www/vhosts/domain.tld/:/tmp/:/proc/meminfo:/proc/cpuinfo:/proc/uptime:/usr/share/dict/words) at /var/www/vhosts/domain.tld/cloud.domain.tld/lib/private/Template/IconsCacher.php#132	
vor 2 Minuten
Error	PHP	Error: file_exists(): open_basedir restriction in effect. File(/img/reply.svg) is not within the allowed path(s): (/var/www/vhosts/domain.tld/:/tmp/:/proc/meminfo:/proc/cpuinfo:/proc/uptime:/usr/share/dict/words) at /var/www/vhosts/domain.tld/cloud.domain.tld/lib/private/Template/IconsCacher.php#132	
vor 2 Minuten
Error	PHP	Error: file_exists(): open_basedir restriction in effect. File(/img/drag-handle.svg) is not within the allowed path(s): (/var/www/vhosts/domain.tld/:/tmp/:/proc/meminfo:/proc/cpuinfo:/proc/uptime:/usr/share/dict/words) at /var/www/vhosts/domain.tld/cloud.domain.tld/lib/private/Template/IconsCacher.php#132	
vor 2 Minuten
Error	PHP	Error: file_exists(): open_basedir restriction in effect. File(/img/answer-long.svg) is not within the allowed path(s): (/var/www/vhosts/domain.tld/:/tmp/:/proc/meminfo:/proc/cpuinfo:/proc/uptime:/usr/share/dict/words) at /var/www/vhosts/domain.tld/cloud.domain.tld/lib/private/Template/IconsCacher.php#132	
vor 2 Minuten
Error	PHP	Error: file_exists(): open_basedir restriction in effect. File(/img/answer-short.svg) is not within the allowed path(s): (/var/www/vhosts/domain.tld/:/tmp/:/proc/meminfo:/proc/cpuinfo:/proc/uptime:/usr/share/dict/words) at /var/www/vhosts/domain.tld/cloud.domain.tld/lib/private/Template/IconsCacher.php#132	
vor 2 Minuten
Error	PHP	Error: file_exists(): open_basedir restriction in effect. File(/img/answer-dropdown.svg) is not within the allowed path(s): (/var/www/vhosts/domain.tld/:/tmp/:/proc/meminfo:/proc/cpuinfo:/proc/uptime:/usr/share/dict/words) at /var/www/vhosts/domain.tld/cloud.domain.tld/lib/private/Template/IconsCacher.php#132	
vor 2 Minuten
Error	PHP	Error: file_exists(): open_basedir restriction in effect. File(/img/answer-multiple.svg) is not within the allowed path(s): (/var/www/vhosts/domain.tld/:/tmp/:/proc/meminfo:/proc/cpuinfo:/proc/uptime:/usr/share/dict/words) at /var/www/vhosts/domain.tld/cloud.domain.tld/lib/private/Template/IconsCacher.php#132	
vor 2 Minuten
Error	PHP	Error: file_exists(): open_basedir restriction in effect. File(/img/answer-checkbox.svg) is not within the allowed path(s): (/var/www/vhosts/domain.tld/:/tmp/:/proc/meminfo:/proc/cpuinfo:/proc/uptime:/usr/share/dict/words) at /var/www/vhosts/domain.tld/cloud.domain.tld/lib/private/Template/IconsCacher.php#132	
vor 2 Minuten
Error	PHP	Error: file_exists(): open_basedir restriction in effect. File(/img/forms.svg) is not within the allowed path(s): (/var/www/vhosts/domain.tld/:/tmp/:/proc/meminfo:/proc/cpuinfo:/proc/uptime:/usr/share/dict/words) at /var/www/vhosts/domain.tld/cloud.domain.tld/lib/private/Template/IconsCacher.php#132	
vor 2 Minuten
Error	PHP	Error: file_exists(): open_basedir restriction in effect. File(/img/rename.svg) is not within the allowed path(s): (/var/www/vhosts/domain.tld/:/tmp/:/proc/meminfo:/proc/cpuinfo:/proc/uptime:/usr/share/dict/words) at /var/www/vhosts/domain.tld/cloud.domain.tld/lib/private/Template/IconsCacher.php#132	
vor 2 Minuten
Error	PHP	Error: file_exists(): open_basedir restriction in effect. File(/img/icon-manual.svg) is not within the allowed path(s): (/var/www/vhosts/domain.tld/:/tmp/:/proc/meminfo:/proc/cpuinfo:/proc/uptime:/usr/share/dict/words) at /var/www/vhosts/domain.tld/cloud.domain.tld/lib/private/Template/IconsCacher.php#132	
vor 2 Minuten
Error	PHP	Error: file_exists(): open_basedir restriction in effect. File(/img/icon-list-up.svg) is not within the allowed path(s): (/var/www/vhosts/domain.tld/:/tmp/:/proc/meminfo:/proc/cpuinfo:/proc/uptime:/usr/share/dict/words) at /var/www/vhosts/domain.tld/cloud.domain.tld/lib/private/Template/IconsCacher.php#132	
vor 2 Minuten
Error	PHP	Error: file_exists(): open_basedir restriction in effect. File(/img/icon-list-down.svg) is not within the allowed path(s): (/var/www/vhosts/domain.tld/:/tmp/:/proc/meminfo:/proc/cpuinfo:/proc/uptime:/usr/share/dict/words) at /var/www/vhosts/domain.tld/cloud.domain.tld/lib/private/Template/IconsCacher.php#132	
vor 2 Minuten
Error	PHP	Error: file_exists(): open_basedir restriction in effect. File(/img/icon-subtasks-hidden.svg) is not within the allowed path(s): (/var/www/vhosts/domain.tld/:/tmp/:/proc/meminfo:/proc/cpuinfo:/proc/uptime:/usr/share/dict/words) at /var/www/vhosts/domain.tld/cloud.domain.tld/lib/private/Template/IconsCacher.php#132	
vor 2 Minuten
Error	PHP	Error: file_exists(): open_basedir restriction in effect. File(/img/icon-subtasks-visible.svg) is not within the allowed path(s): (/var/www/vhosts/domain.tld/:/tmp/:/proc/meminfo:/proc/cpuinfo:/proc/uptime:/usr/share/dict/words) at /var/www/vhosts/domain.tld/cloud.domain.tld/lib/private/Template/IconsCacher.php#132	
vor 2 Minuten
Error	PHP	Error: file_exists(): open_basedir restriction in effect. File(/img/icon-alphabetically.svg) is not within the allowed path(s): (/var/www/vhosts/domain.tld/:/tmp/:/proc/meminfo:/proc/cpuinfo:/proc/uptime:/usr/share/dict/words) at /var/www/vhosts/domain.tld/cloud.domain.tld/lib/private/Template/IconsCacher.php#132	
vor 2 Minuten
Error	PHP	Error: file_exists(): open_basedir restriction in effect. File(/img/icon-task-star.svg) is not within the allowed path(s): (/var/www/vhosts/domain.tld/:/tmp/:/proc/meminfo:/proc/cpuinfo:/proc/uptime:/usr/share/dict/words) at /var/www/vhosts/domain.tld/cloud.domain.tld/lib/private/Template/IconsCacher.php#132	
vor 2 Minuten
Error	PHP	Error: file_exists(): open_basedir restriction in effect. File(/img/icon-list.svg) is not within the allowed path(s): (/var/www/vhosts/domain.tld/:/tmp/:/proc/meminfo:/proc/cpuinfo:/proc/uptime:/usr/share/dict/words) at /var/www/vhosts/domain.tld/cloud.domain.tld/lib/private/Template/IconsCacher.php#132	
vor 2 Minuten
Error	PHP	Error: file_exists(): open_basedir restriction in effect. File(/img/icon-due.svg) is not within the allowed path(s): (/var/www/vhosts/domain.tld/:/tmp/:/proc/meminfo:/proc/cpuinfo:/proc/uptime:/usr/share/dict/words) at /var/www/vhosts/domain.tld/cloud.domain.tld/lib/private/Template/IconsCacher.php#132	
vor 2 Minuten
Error	PHP	Error: file_exists(): open_basedir restriction in effect. File(/img/icon-start.svg) is not within the allowed path(s): (/var/www/vhosts/domain.tld/:/tmp/:/proc/meminfo:/proc/cpuinfo:/proc/uptime:/usr/share/dict/words) at /var/www/vhosts/domain.tld/cloud.domain.tld/lib/private/Template/IconsCacher.php#132	
vor 2 Minuten
Error	PHP	Error: file_exists(): open_basedir restriction in effect. File(/img/color-picker.svg) is not within the allowed path(s): (/var/www/vhosts/domain.tld/:/tmp/:/proc/meminfo:/proc/cpuinfo:/proc/uptime:/usr/share/dict/words) at /var/www/vhosts/domain.tld/cloud.domain.tld/lib/private/Template/IconsCacher.php#132	
vor 2 Minuten
Error	PHP	Error: file_exists(): open_basedir restriction in effect. File(/img/timezone.svg) is not within the allowed path(s): (/var/www/vhosts/domain.tld/:/tmp/:/proc/meminfo:/proc/cpuinfo:/proc/uptime:/usr/share/dict/words) at /var/www/vhosts/domain.tld/cloud.domain.tld/lib/private/Template/IconsCacher.php#132	
vor 2 Minuten
Error	PHP	Error: file_exists(): open_basedir restriction in effect. File(/img/repeat.svg) is not within the allowed path(s): (/var/www/vhosts/domain.tld/:/tmp/:/proc/meminfo:/proc/cpuinfo:/proc/uptime:/usr/share/dict/words) at /var/www/vhosts/domain.tld/cloud.domain.tld/lib/private/Template/IconsCacher.php#132	
vor 2 Minuten
Error	PHP	Error: file_exists(): open_basedir restriction in effect. File(/img/reminder.svg) is not within the allowed path(s): (/var/www/vhosts/domain.tld/:/tmp/:/proc/meminfo:/proc/cpuinfo:/proc/uptime:/usr/share/dict/words) at /var/www/vhosts/domain.tld/cloud.domain.tld/lib/private/Template/IconsCacher.php#132	
vor 2 Minuten
Error	PHP	Error: file_exists(): open_basedir restriction in effect. File(/img/briefcase.svg) is not within the allowed path(s): (/var/www/vhosts/domain.tld/:/tmp/:/proc/meminfo:/proc/cpuinfo:/proc/uptime:/usr/share/dict/words) at /var/www/vhosts/domain.tld/cloud.domain.tld/lib/private/Template/IconsCacher.php#132	
vor 2 Minuten
Error	PHP	Error: file_exists(): open_basedir restriction in effect. File(/img/eye.svg) is not within the allowed path(s): (/var/www/vhosts/domain.tld/:/tmp/:/proc/meminfo:/proc/cpuinfo:/proc/uptime:/usr/share/dict/words) at /var/www/vhosts/domain.tld/cloud.domain.tld/lib/private/Template/IconsCacher.php#132	
vor 2 Minuten
Error	PHP	Error: file_exists(): open_basedir restriction in effect. File(/img/new-calendar-with-task-list.svg) is not within the allowed path(s): (/var/www/vhosts/domain.tld/:/tmp/:/proc/meminfo:/proc/cpuinfo:/proc/uptime:/usr/share/dict/words) at /var/www/vhosts/domain.tld/cloud.domain.tld/lib/private/Template/IconsCacher.php#132	
vor 2 Minuten
Error	PHP	Error: file_exists(): open_basedir restriction in effect. File(/img/new-calendar.svg) is not within the allowed path(s): (/var/www/vhosts/domain.tld/:/tmp/:/proc/meminfo:/proc/cpuinfo:/proc/uptime:/usr/share/dict/words) at /var/www/vhosts/domain.tld/cloud.domain.tld/lib/private/Template/IconsCacher.php#132	
vor 2 Minuten
Error	PHP	Error: file_exists(): open_basedir restriction in effect. File(/img/embed.svg) is not within the allowed path(s): (/var/www/vhosts/domain.tld/:/tmp/:/proc/meminfo:/proc/cpuinfo:/proc/uptime:/usr/share/dict/words) at /var/www/vhosts/domain.tld/cloud.domain.tld/lib/private/Template/IconsCacher.php#132	
vor 2 Minuten
Error	PHP	Error: file_exists(): open_basedir restriction in effect. File(/img/view-list.svg) is not within the allowed path(s): (/var/www/vhosts/domain.tld/:/tmp/:/proc/meminfo:/proc/cpuinfo:/proc/uptime:/usr/share/dict/words) at /var/www/vhosts/domain.tld/cloud.domain.tld/lib/private/Template/IconsCacher.php#132	
vor 2 Minuten
Error	PHP	Error: file_exists(): open_basedir restriction in effect. File(/img/view-week.svg) is not within the allowed path(s): (/var/www/vhosts/domain.tld/:/tmp/:/proc/meminfo:/proc/cpuinfo:/proc/uptime:/usr/share/dict/words) at /var/www/vhosts/domain.tld/cloud.domain.tld/lib/private/Template/IconsCacher.php#132	
vor 2 Minuten
Error	PHP	Error: file_exists(): open_basedir restriction in effect. File(/img/view-day.svg) is not within the allowed path(s): (/var/www/vhosts/domain.tld/:/tmp/:/proc/meminfo:/proc/cpuinfo:/proc/uptime:/usr/share/dict/words) at /var/www/vhosts/domain.tld/cloud.domain.tld/lib/private/Template/IconsCacher.php#132	
vor 2 Minuten
Error	PHP	Error: file_exists(): open_basedir restriction in effect. File(/img/view-module.svg) is not within the allowed path(s): (/var/www/vhosts/domain.tld/:/tmp/:/proc/meminfo:/proc/cpuinfo:/proc/uptime:/usr/share/dict/words) at /var/www/vhosts/domain.tld/cloud.domain.tld/lib/private/Template/IconsCacher.php#132	
vor 2 Minuten
Error	PHP	Error: file_exists(): open_basedir restriction in effect. File(/img/rightarrow.svg) is not within the allowed path(s): (/var/www/vhosts/domain.tld/:/tmp/:/proc/meminfo:/proc/cpuinfo:/proc/uptime:/usr/share/dict/words) at /var/www/vhosts/domain.tld/cloud.domain.tld/lib/private/Template/IconsCacher.php#132	
vor 2 Minuten
Error	PHP	Error: file_exists(): open_basedir restriction in effect. File(/img/leftarrow.svg) is not within the allowed path(s): (/var/www/vhosts/domain.tld/:/tmp/:/proc/meminfo:/proc/cpuinfo:/proc/uptime:/usr/share/dict/words) at /var/www/vhosts/domain.tld/cloud.domain.tld/lib/private/Template/IconsCacher.php#132
</details>

Pretty much all there paths are inside the nextcloud folder. As you can see, this folder is part of the allowed paths

### PR DESCRIPTION
**Pull request by Hinyka**
Fix https://github.com/nextcloud/server/issues/27759 with Lots of Error: file_exists(): open_basedir restriction in effect. in Log

Signed-off-by: Karel Hink <info@karelhink.cz>

### CODE REVIEW

**Comment by kesselb** [comment]
Thanks for sending a patch :+1:

File: lib/private/Template/IconsCacher.php [hunk 1]
```diff
@@ -170,7 +170,10 @@ private function parseUrl($url): array {
 		} elseif (\strpos($url, $base) === 0) {
 			if (\preg_match('/([A-z0-9\_\-]+)\/([a-zA-Z0-9-_\~\/\.\=\:\;\+\,]+)\?color=([0-9a-fA-F]{3,6})/', $cleanUrl, $matches)) {
 				[,$app,$cleanUrl, $color] = $matches;
-				$location = \OC_App::getAppPath($app) . '/img/' . $cleanUrl . '.svg';
+				$appPath = \OC_App::getAppPath($app);
+				if ($appPath !== false) {
```
Comment (PVince81) [inline lib/private/Template/IconsCacher.php:174]: so when $appPath is false, $location will be an empty string

did you check whether an empty location could cause other issues ?

Comment (Hinyka) [reply lib/private/Template/IconsCacher.php:174]: Empty `$location` is checked here and is the purpose in this case
https://github.com/nextcloud/server/blob/afb0b379d907744c7baa1ad2a98da818c86f118f/lib/private/Template/IconsCacher.php#L128-L138

I didn't find any errors.

**Review by PVince81 — APPROVED** [review]
:+1:

**Review by blizzz — APPROVED** [review]

**Pull request merged**